reco_utils/setup.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from setuptools import setup, find_packages
+from os import path
+
+
+VERSION = __import__('__init__').VERSION
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='reco_utils',
+    version=VERSION,
+    description='Recommender System Utilities',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    url='https://github.com/microsoft/recommenders',
+    author='RecoDev Team at Microsoft',
+    author_email='RecoDevTeam@service.microsoft.com',
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Intended Audience :: Developers',
+        'Intended Audience :: Science/Research',
+        'Topic :: Scientific/Engineering',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+    ],
+    keywords='recommendations recommenders recommender system engine machine learning python spark gpu',
+    packages=find_packages(),
```

Comment:
hey can you check whether this takes the tests and the notebooks folder? not sure if we have to do:
```
    packages=find_packages(
        exclude=[
            "*test*",
            "*notebooks",
        ]
    ),
```

Reply:
Yeah, good catch, it does indeed pull in tests and scripts since those dirs have __init__.py files (which they probably shouldn't)

Reply:
I'll force the setup to move to the reco_utils folder, which will avoid the top level scripts and tests dirs regardless of where the install is invoked from

Reply:
Testing the pip install
What about automated testing to validate pip install functionality?